src/array/show.jl

```diff
@@ -73,7 +73,7 @@ function print_name(io::IO, name)
     end
 end
 
-Base.print_matrix(io::IO, A::AbstractDimArray) = _print_matrix(io, parent(A), lookup(A))
+# Base.print_matrix(io::IO, A::AbstractDimArray) = _print_matrix(io, parent(A), lookup(A))
```

Comment:
Why is this commented out instead of removed?

Reply:
accident, that fixed the coverage too